rules/__tests__/no-disabled-tests.test.js

```diff
@@ -3,7 +3,11 @@
 const RuleTester = require('eslint').RuleTester;
 const rule = require('../no-disabled-tests');
 
-const ruleTester = new RuleTester();
+const ruleTester = new RuleTester({
+  parserOptions: {
+    sourceType: 'module',
```

Comment:
This allows for writing examples that use ES modules (i.e. `import` statements).